vllm/model_executor/guided_decoding/guided_fields.py

```diff
@@ -3,6 +3,7 @@
 
 from pydantic import BaseModel
 
+# Nick leans towards ripping out so we don't have duplication
```

Comment:
At a minimum we should deprecate for a version.

Reply:
Yup, I'll stick a warning on!